BUILDING.md

```diff
@@ -20,7 +20,7 @@ If in doubt, use the most recent stable version of each build tool.
     `PERL_EXECUTABLE`.
     * To build without Perl (not recommended) see [this section.](#using-pre-generated-build-files)
 
-  * [Go](https://golang.org/dl/) 1.18 or later is required. If not found by
+  * [Go](https://golang.org/dl/) 1.20 or later is required. If not found by
```

Comment:
go 1.20 is required for running the tests, but I believe the project can still be built using go 1.18. Perhaps we could set go 1.20 as the minimum only when `BUILD_TESTING` is enabled?

These other places would also need to be updated:
* https://github.com/aws/aws-lc/blob/b5e3681ff4dd984a5bab223c676752e954c1b3f7/cmake/go.cmake#L25
* https://github.com/aws/aws-lc/blob/b5e3681ff4dd984a5bab223c676752e954c1b3f7/go.mod#L4-L3